scripts/install/install_unix.sh

```diff
@@ -640,17 +647,32 @@ install_package()
     succeeded
   fi
 
-  if [ "$(systemctl is-enabled observiq-otel-collector)" = "enabled" ]; then
-    # The unit is already enabled; It may be running, too, if this was an upgrade.
-    # We'll want to restart, which will start it if it wasn't running already,
-    # and restart in the case that this was an upgrade on a running agent.
-    info "Restarting service..."
-    systemctl restart observiq-otel-collector
-    succeeded
+  if [ $SVC_PRE == "systemctl" ]; then
+    if [ "$(systemctl is-enabled observiq-otel-collector)" = "enabled" ]; then
+      # The unit is already enabled; It may be running, too, if this was an upgrade.
+      # We'll want to restart, which will start it if it wasn't running already,
+      # and restart in the case that this was an upgrade on a running agent.
+      info "Restarting service..."
+      systemctl restart observiq-otel-collector
+      succeeded
+    else
+      info "Enabling service..."
+      systemctl enable --now observiq-otel-collector > /dev/null 2>&1 || error_exit "$LINENO" "Failed to enable service"
+      succeeded
+    fi
   else
-    info "Enabling service..."
-    systemctl enable --now observiq-otel-collector > /dev/null 2>&1 || error_exit "$LINENO" "Failed to enable service"
-    succeeded
+    if [[ $(service observiq-otel-collector status) = *running* ]]; then
+      # The service is running.
+      # We'll want to restart.
+      info "Restarting service..."
+      service observiq-otel-collector restart
+      succeeded
+    else
+      info "Enabling and starting service..."
+      chkconfig observiq-otel-collector on
+      service observiq-otel-collector start
+      succeeded
+    fi
```

Comment:
Output of the install script looks like this:
```
  Enabling and starting service...
    Succeeded!init.d/observiq-otel-collector:              [  OK  ]
```

I think we want to swallow the output of whatever puts `init.d/observiq-otel-collector:              [  OK  ]` to stdout